src/frontend/web_application/package.json

```diff
@@ -63,6 +63,7 @@
     "iron": "^4.0.4",
     "jquery": "^3.1.1",
     "locale": "^0.1.0",
+    "lodash": "^4.17.4",
```

Comment:
Can you share a shrinkwrap of the application's dependency tree?

If `lodash` is not already there, you might want to use the standalone lodash throttle package `https://www.npmjs.com/package/lodash.throttle`